ui-admin/src/util/juniperSurveyUtils.tsx

```diff
@@ -141,3 +142,15 @@ export function questionFromRawText(rawText: string): QuestionObj {
 ReactQuestionFactory.Instance.registerQuestion('addressvalidation', props => {
   return React.createElement(SurveyQuestionAddressValidation, props)
 })
+
+/**
+ * Takes in an I18nSurveyElement and either returns the default value (if I18nSurveyElement is in the
+ * internationalized format), or the string itself (if I18nSurveyElement is a simple string).'
+ * Once the survey designer is fully
+ */
+export const getI18nSurveyElement = (element?: I18nSurveyElement, language?: string) => {
```

Comment:
making the argument default cleans up the ternary below
```suggestion
export const getI18nSurveyElement = (element?: I18nSurveyElement, language = 'default') => {
```

Reply:
I might also rename this to `i18nSurveyText` to be a bit shorter